### ISSUE TITLE
PortraitNet Model

### ISSUE THREAD

**Issue by shivSD** [anilsathyan7/Portrait-Segmentation#31]
@anilsathyan7  thanks for the great repo. 
Question PortraitNet model that you have used in this repo. Did you train the model from scratch or directly took the author's pre-trained model. I'm trying to train from the scratch so far unsuccessful, the training loss doesn't decrease after a few hundred epochs.  Any suggestions on this would be useful.

**Comment by anilsathyan7** [comment]
As mentioned in the section - Techniques For Improving Robustness, Accuracy and Speed, i trained the whole network from scratch using AISegment dataset. I had to modify the original code to train the model from scratch. It seems to use syntax/fucntions from an older version of tensorflow and pytorch.

Also make sure you use the correct dataset and training configuration, before trainin the model.
